powerlevel9k.zsh-theme

```diff
@@ -1004,10 +1004,30 @@ prompt_ssh() {
 # Status: return code if verbose, otherwise just an icon if an error occurred
 set_default POWERLEVEL9K_STATUS_VERBOSE true
 set_default POWERLEVEL9K_STATUS_OK_IN_NON_VERBOSE false
+set_default POWERLEVEL9K_STATUS_SHOW_PIPESTATUS false
```

Comment:
Nice work using our built-ins, especially for a first-time contributor =)

So, actually, I think I would like for this to default to being `true` - I think it's a really great improvement. You'll also need to update the docs in the README to reflect this.